db/migrate/20190225133654_add_ignored_users_table.rb

```diff
@@ -0,0 +1,12 @@
+class AddIgnoredUsersTable < ActiveRecord::Migration[5.2]
+  def change
+    create_table :ignored_users, force: true do |t|
+      t.integer :user_id, null: false
```

Comment:
Do we also need to add a foreign key?

Reply:
Yes, I wonder what's our stance in general on foreign keys. Different Rails codebases get to be opinionated about using foreign keys at all in their databases (they have their reasons).

Do we default on using them? Or only when it makes sense?

Reply:
> Or only when it makes sense?

This ^^

Reply:
Makes sense. IMO, I see in this case we might not need to add a foreign key. Looking at another table with similar usage and context `muted_users`, it was not need in that case either.